forge/src/multi_runner.rs

```diff
@@ -220,8 +220,8 @@ mod tests {
         let mut runner = runner(evm);
         let results = runner.test(&Filter::new(".*", ".*")).unwrap();
 
-        // 6 contracts being built
-        assert_eq!(results.keys().len(), 7);
+        // 8 contracts being built
+        assert_eq!(results.keys().len(), 8);
```

Comment:
think this is because of the regression mentioned in 4bdff194667accbb06a83e307c7e3cb05b0b4282

Reply:
my contract isnt abstract so it should still increment by 1, no?

Reply:
mb :facepalm:

Reply:
The regression should be unrelated, this seems correct, there's a new contract added `BadSetup` so makes sense that there's 1 more key.